README.md

```diff
@@ -19,7 +19,7 @@ The code for Habitat itself is in the
 
 ## License
 
-Copyright (c) 2016 Chef Software Inc. and/or applicable contributors
+Copyright (c) 2016-2017 Chef Software Inc. and/or applicable contributors
```

Comment:
I learned from @adamhjk that you don't actually have to update these; the only year you need in a copyright notice is the year the work was created, so he said that lawyers told him.